service/network/allocator_test.go

```diff
@@ -113,145 +133,3 @@ func Test_Allocator(t *testing.T) {
 		})
 	}
 }
-
-func Test_Allocator_Locking(t *testing.T) {
```

Comment:
For the record. Tuomas dropped these in the other package, because we figured out they are flapping and we couldn't write them properly within a reasonable amount of time. I drop them here as they make the build fail. I can also not reproduce the issue locally as they only appear to fail on Circle. I may have a think about it later.